src/main/webapp/jsp/module/purap/IWant.jsp

```diff
@@ -83,6 +83,10 @@
 
         <purap:iWantNotes attachmentTypesValuesFinderClass="${documentEntry.attachmentTypesValuesFinderClass}"
                           defaultOpen="true"/>
+        
+        <c:if test="${not isRegularStep}">

```

Comment:
Yes, I agree it would be good to update this to "!" for consistency with the other two lines; I'll make the change. Thanks for checking.